server/src/main/java/org/elasticsearch/cluster/routing/allocation/WriteLoadConstraintSettings.java

```diff
@@ -23,6 +24,7 @@
 public class WriteLoadConstraintSettings {
 
     private static final String SETTING_PREFIX = "cluster.routing.allocation.write_load_decider.";
+    private static final FeatureFlag WRITE_LOAD_DECIDER_ENABLED_FF = new FeatureFlag("write_load_decider_enabled");
```

Comment:
I'd probably get rid of 'enabled' in both names, as it's redundant, but doesn't really matter.